
From: Geert Uytterhoeven <geert@linux-m68k.org>

Kill remainings of the DMI Resolver support code that got removed somewhere
between 2.0 and 2.2.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/video/amifb.c |   13 -------------
 1 files changed, 13 deletions(-)

diff -puN drivers/video/amifb.c~amiga-frame-buffer-kill-obsolete-dmi-resolver-code drivers/video/amifb.c
--- 25/drivers/video/amifb.c~amiga-frame-buffer-kill-obsolete-dmi-resolver-code	2004-10-02 18:41:02.228465664 -0700
+++ 25-akpm/drivers/video/amifb.c	2004-10-02 18:41:02.234464752 -0700
@@ -2268,19 +2268,6 @@ int __init amifb_init(void)
 		return -ENXIO;
 
 	/*
-	 * TODO: where should we put this? The DMI Resolver doesn't have a
-	 *	 frame buffer accessible by the CPU
-	 */
-
-#ifdef CONFIG_GSP_RESOLVER
-	if (amifb_resolver){
-		custom.dmacon = DMAF_MASTER | DMAF_RASTER | DMAF_COPPER |
-				DMAF_BLITTER | DMAF_SPRITE;
-		return 0;
-	}
-#endif
-
-	/*
 	 * We request all registers starting from bplpt[0]
 	 */
 	if (!request_mem_region(CUSTOM_PHYSADDR+0xe0, 0x120,
_
